### PR TITLE
[CP] LRSD-7202 - Add missing translations

### PR DESCRIPTION
**Pull request by abfranca**
https://liferay.atlassian.net/browse/LRSD-7202

### CODE REVIEW

**Comment by abfranca** [comment]
ci:test:sf

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/utils/constants/filterOptions.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const FILTER_MAP: {[key: string]: string} = {
 
 export const FILTER_OPTIONS: IProps = {
 	[JiraEnum.AFFECTED_VERSIONS]: ['2024.Q4', '2024.Q3', '2024.Q2', '2024.Q1'],
-	[JiraEnum.CATEGORIES]: ['Paas', 'Saas', 'Self-Hosted', 'Docker'],
+	[JiraEnum.CATEGORIES]: ['PaaS', 'SaaS', 'On Premise', 'Docker'],
```
Comment (kylembischof) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/utils/constants/filterOptions.ts:26]: Should we sort these? @ryanschuhler thoughts?

Comment (abfranca) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/utils/constants/filterOptions.ts:26]: this is the order that will appear in the frontend, on the figma we have Docker > Self-Hosted (On Premise now) > PaaS > SaaS
i think we should change the order, just not sure if we should use sorted or the figma order (which kinda makes sense concrete > abstract)

Comment (kylembischof) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/utils/constants/filterOptions.ts:26]: I asked Tiffany on the ticket.

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/pages/SecurityVulnerabilitiesItem/SecurityVulnerabilitiesItem.tsx [hunk 2]
```diff
@@ -212,7 +212,7 @@ const SecurityVulnerabilitiesItem = () => {
 						{jiraIssue[JiraEnum.FIELDS]?.[JiraEnum.CATEGORIES] && (
 							<div className="mb-4">
 								<h5 className="text-neutral-10">
-									{i18n.translate('categories')}
+									{i18n.translate('category')}
```
Comment (ryanschuhler) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/pages/SecurityVulnerabilitiesItem/SecurityVulnerabilitiesItem.tsx:215]: are we sure we want these singular? feels odd when there are multiple listed right below it. If we do want it singular, should we also change Fix Versions to Fix Version?

Comment (kylembischof) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/pages/SecurityVulnerabilitiesItem/SecurityVulnerabilitiesItem.tsx:215]: I think they should match the singular that Tiffany wanted on the search page. So I think this makes sense.

Comment (ryanschuhler) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/pages/SecurityVulnerabilitiesItem/SecurityVulnerabilitiesItem.tsx:215]: okay, so then also update Fix Versions to be singular?

Comment (kylembischof) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/pages/SecurityVulnerabilitiesItem/SecurityVulnerabilitiesItem.tsx:215]: yeah, lets go singular with that one also, to stay consistent.

Comment (abfranca) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/pages/SecurityVulnerabilitiesItem/SecurityVulnerabilitiesItem.tsx:215]: Done!

**Comment by ryanschuhler** [comment]
ci:test

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/utils/constants/filterOptions.ts (hunk 1 above)
Comment (kylembischof) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/utils/constants/filterOptions.ts:26]: yes lets sort both Category and Issue Classification filters. Severity stays the same sorted in order of severity.

Comment (abfranca) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/utils/constants/filterOptions.ts]: Done!

**Comment by ryanschuhler** [comment]
looks good! https://github.com/brianchandotcom/liferay-portal/pull/156955

**Pull request closed without merging**